src/export/digitalPlanning/model.ts

```diff
@@ -352,9 +370,8 @@ export class DigitalPlanning {
         ),
       },
       planning: this.getPlanningConstraints(),
-      // Only include the 'boundary' key in cases where we have digital data, not an uploaded location plan
-      ...(this.passport.data?.["property.boundary.site"] && {
-        boundary: this.getBoundary(),
+      ...(this.passport.data?.["property.boundary.title"] && {
+        boundary: this.getPropertyBoundary(),
```

Comment:
This is an optimistic mapping & won't be populated until this merges https://github.com/theopensystemslab/planx-new/pull/2561